webrender/src/device/gfx/device.rs

```diff
@@ -562,10 +564,10 @@ impl<B: hal::Backend> Device<B> {
 
     pub(crate) fn recreate_swapchain(&mut self, window_size: Option<(i32, i32)>) -> DeviceIntSize {
         self.device.wait_idle().unwrap();
-
-        for (_id, program) in self.programs.drain() {
+        // We got 'Program not found' after recreating the swapchain
```

Comment:
I think this happens because the debug programs are in the `DebugRenderer`, and the Program not found comes from there. If we go with the wait for the resize event if something goes wrong with the present or acquire_image, then this should fixed by that, since `Renderer`'s `resize` will take care of that.